packages/cli/src/services/redis/redis.types.ts

```diff
@@ -0,0 +1,19 @@
+export type RedisClient = n8nRedisClient | BullRedisClient;
+
+/**
+ * Redis client used by n8n.
+ *
+ * - `subscriber(n8n)` to listen for messages from scaling mode communication channels
+ * - `publisher(n8n)` to send messages into scaling mode communication channels
+ * - `cache(n8n)` for caching operations (variables, resource ownership, etc.)
+ */
+type n8nRedisClient = 'subscriber(n8n)' | 'publisher(n8n)' | 'cache(n8n)';
```

Comment:
TS types should start with a capital letter, even if it's for a name that is normally written with a lowercase letter